### PR TITLE
Stop server-script URLs when deleted from entity properties (case 11244)

### PR DESCRIPTION
**Pull request by SimonWalton-HiFi**
This is working correctly in my testing, but I've only been able to test simple scenarios.

The case has these repro steps:

>Steps to Reproduce:
>1)Open High Fidelity PR12110
>2)Select Create
>3)Select Text
>4)enter a Server Script URL
>5)once the script is running delete the URL from the properties tab 
>6)Script continues to run 

>Expected Result:Script stops when the URL is deleted 

>Current Result:Script does not stop when the URL is deleted

The case also has a suitable test.js script.

https://highfidelity.manuscript.com/f/cases/11244/

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7552/app_12508_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7552/HighFidelity-Beta-PR12508-40f0e63af0cd5d09f7f7e240e159c7d7ac4eca88.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7552/HighFidelity-Beta-PR12508-40f0e63af0cd5d09f7f7e240e159c7d7ac4eca88.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7564/app_12508_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7564/HighFidelity-Beta-PR12508-051d4c0d3567dc5f390d288d8f5ab30c9a1b0517.dmg

**Comment by ZappoMan** [comment]
jenkins test this please

File: libraries/model-networking/src/model-networking/TextureCache.h [hunk 1]
```diff
@@ -133,6 +133,8 @@ public slots:
 
 using NetworkTexturePointer = QSharedPointer<NetworkTexture>;
 
+Q_DECLARE_METATYPE(QWeakPointer<NetworkTexture>)
```
Comment (ZappoMan) [inline libraries/model-networking/src/model-networking/TextureCache.h]: why is this change in this PR? this seems like a weird change to be included in this PR

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7642/app_12508_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7642/HighFidelity-Beta-PR12508-051d4c0d3567dc5f390d288d8f5ab30c9a1b0517.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7642/HighFidelity-Beta-PR12508-051d4c0d3567dc5f390d288d8f5ab30c9a1b0517.exe

**Comment by qlimnavarro** [comment]
Currently Testing.

**Comment by qlimnavarro** [comment]
Pass - @conklin94122

**Pull request merged**